examples/property_prediction/pretrain_gnns/chem/pretrain_masking.py

```diff
@@ -0,0 +1,214 @@
+    parser.add_argument('--emb_dim', type=int, default=300,
+                        help='embedding dimensions (default: 300)')
+    parser.add_argument('--dropout_ratio', type=float, default=0,
+                        help='dropout ratio (default: 0)')
+    parser.add_argument('--mask_rate', type=float, default=0.15,
+                        help='dropout ratio (default: 0.15)')
+    parser.add_argument('--mask_edge', type=int, default=0,
+                        help='whether to mask edges or not together with atoms')
+    parser.add_argument('--JK', type=str, default="last",
+                        help='how the node features are combined across layers. last, sum, max or concat')
+    parser.add_argument('--dataset', type=str, default='zinc_standard_agent',
+                        help='root directory of dataset for pretraining')
+    parser.add_argument('--output_model_file', type=str, default='', help='filename to output the model')
+    parser.add_argument('--seed', type=int, default=0, help="Seed.")
+    parser.add_argument('--num_workers', type=int, default=8, help='number of workers for dataset loading')
+    args = parser.parse_args()
+    print(args)
+
+    torch.manual_seed(args.seed)
+    if torch.cuda.is_available():
+        torch.cuda.manual_seed_all(args.seed)
+
+    device = torch.device("cuda:" + str(args.device)) if torch.cuda.is_available() else torch.device("cpu")
+
+    if args.zinc_standard_agent is not 'zinc_standard_agent':
+        raise ValueError('Dataset should be zinc_standard_agent.')
+    df = pd.read_csv('./zinc.csv')
+    # df = pd.read_csv('./test_tiny_data_zinc.txt')
+    atom_featurizer = PretrainAtomFeaturizer()
+    bond_featurizer = PretrainBondFeaturizer()
+    dataset = PretrainMaskingMoleculeCSVDataset(df=df,
+                                                smiles_to_graph=partial(smiles_to_bigraph, add_self_loop=True),
+                                                node_featurizer=atom_featurizer,
+                                                edge_featurizer=bond_featurizer,
+                                                smiles_column='smiles')
+
+    train_dataloader = DataLoader(dataset=dataset,
+                                  batch_size=args.batch_size,
+                                  shuffle=True,
+                                  num_workers=args.num_workers,
+                                  collate_fn=partial(collate_masking, args=args))
+
+    # 118 atom number plus 1 masked node type = 119
+    # 4 bond type plus self-loop plus 1 masked node type = 6
+    model = GINMaskingModel(num_node_emb_list=[119, 4],
+                            num_edge_emb_list=[6, 3],
+                            num_layers=args.num_layer,
+                            emb_dim=args.emb_dim,
+                            JK=args.JK,
+                            dropout=args.dropout_ratio)
+    node_linear = torch.nn.Linear(args.emb_dim, 119)
```

Comment:
Since you already did `import torch.nn as nn`, you can replace `torch.nn.Linear` with `nn.Linear`.

Reply:
ty